<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_input">
      <property name="text">
       <string>Please provide Text here:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="text_input"/>
    </item>
    <item>
     <widget class="QLabel" name="label_result">
      <property name="text">
       <string>Here is your result:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_result"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons">
      <item>
       <widget class="QPushButton" name="pb_check">
        <property name="text">
         <string>check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_reset">
        <property name="text">
         <string>reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_close">
        <property name="text">
         <string>close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_close</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
